x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/Security.java

```diff
@@ -422,6 +423,9 @@ protected XPackLicenseState getLicenseState() {
                 switch (output) {
                     case LoggingAuditTrail.NAME:
                         auditTrails.add(new LoggingAuditTrail(settings, clusterService, threadPool));
+                        // also enabling the deprecated format. To disable it, remove it's associated
+                        // appender in the log4j2.properties file
+                        auditTrails.add(new DeprecatedLoggingAuditTrail(settings, clusterService, threadPool));
```

Comment:
we should have documentation about how to disable this. I am fine if you want to handle this separately, but please open up a issue if that is what you choose to do